src/handlers/mozilla-auth0.js

```diff
@@ -218,31 +218,26 @@ class Handler {
     user.addRole('everybody');
 
     const mozGroupPrefix = 'mozilliansorg_';
-    const mozGroups = [];
-    const ldapGroups = [];
+    const hrisGroupPrefix = 'hris_';
```

Comment:
Yeah, so if Moz adds another source of user information, we're going to interpret it as an LDAP group until it's added to the list here.  I think that's OK, though -- presumably that LDAP group won't also exist or have any roles configured.

Reply:
Yeah, I noted the lack of forward compatibility...

I think we can push a little for what person API might shape up to be..